unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendRequestHUD/SendFriendRequestHUDController.cs

```diff
@@ -75,28 +78,51 @@ private void Show(string recipient)
 
         private async UniTaskVoid SendAsync()
         {
+            hideCancellationToken?.Cancel();
+            hideCancellationToken = new CancellationTokenSource();
+
             view.ShowPendingToSend();
 
             try
             {
+                sendInProgress = true;
+
                 await friendsController.RequestFriendshipAsync(recipientId, messageBody)
                                        .Timeout(TimeSpan.FromSeconds(10));
 
                 socialAnalytics.SendFriendRequestSent(userProfileBridge.GetOwn().userId, recipientId, messageBody.Length,
                     (PlayerActionSource)dataStore.HUDs.sendFriendRequestSource.Get());
 
                 view.ShowSendSuccess();
+                sendInProgress = false;
+
+                await UniTask.Delay(2000, cancellationToken: hideCancellationToken.Token);
```

Comment:
Let's create a consant for the delay time

Reply:
done